### PR TITLE
[feature/OPT-933] to [feature/OPT-792]

### PR DESCRIPTION
**Pull request by daFont-iriusrisk**

### CODE REVIEW

File: tests/integration/api/controllers/iac/cloudformation/test_otm_controller_iac_cloudformation.py [hunk 1]
```diff
@@ -65,10 +67,10 @@ def test_create_otm_ok(self):
         assert '"trustZones": ' in response.text
         assert '"components": ' in response.text
 
-    @mark.parametrize('project_id,project_name,cft_filename,cft_mimetype,mapping_filename,error_type',
-                      [uc_a, uc_b, uc_c, uc_d, uc_e, uc_f, uc_h, uc_i, uc_j, uc_k])
+    @mark.parametrize('project_id,project_name,cft_filename,cft_mimetype,mapping_filename,default_mapping_file,error_type',
+                      [uc_m])
```
Comment (jmgarcia-iriusrisk) [inline tests/integration/api/controllers/iac/cloudformation/test_otm_controller_iac_cloudformation.py:71]: The dataset: 
[uc_a, uc_b, uc_c, uc_d, uc_e, uc_f, uc_h, uc_i, uc_j, uc_k, uc_l] are missing, is it correct?

File: tests/integration/api/controllers/iac/terraform/test_otm_controller_iac_terraform.py [hunk 2]
```diff
@@ -78,10 +79,10 @@ def test_create_otm_ok_all_line_breaks(self, filename: str, break_line: str):
         assert '"trustZones": ' in response.text
         assert '"components": ' in response.text
 
-    @mark.parametrize('project_id,project_name,cft_filename,cft_mimetype,mapping_filename,error_type',
+    @mark.parametrize('project_id,project_name,cft_filename,cft_mimetype,mapping_filename,default_mapping_file,error_type',
```
Comment (jmgarcia-iriusrisk) [inline tests/integration/api/controllers/iac/terraform/test_otm_controller_iac_terraform.py:82]: The dataset:
[uc_l, uc_m] are missing, is it correct?

File: startleft/startleft/api/controllers/iac/iac_create_otm_controller.py [hunk 3]
```diff
@@ -40,11 +42,33 @@ def iac(iac_file: List[UploadFile]
...
+        raise MappingFileNotValidError("Error processing mapping file", msg, msg)
+
+    if mapping_file:
+        with mapping_file.file as f:
+            mapping_data_list.append(f.read())
+
+    if default_mapping_file:
+        with default_mapping_file.file as f:
+            mapping_data_list.append(f.read())
+    return mapping_data_list
```
Comment (jmgarcia-iriusrisk) [inline startleft/startleft/api/controllers/iac/iac_create_otm_controller.py:74]: It may be useful to have a unit test for this method.

**Comment by daFont-iriusrisk** [comment]
PR updated with changes proposal by @jmgarcia-iriusrisk, thank you so much! 😃

**Review by jmgarcia-iriusrisk — APPROVED** [review]

**Pull request merged**